.github/workflows/test.yml

```diff
@@ -1,23 +1,23 @@
-name: Test LocalGov Drupal
+name: Execute Drupal's run-tests.sh for this branch
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
-  test:
-
+  instance-1:
```

Comment:
Can this be restored to "test" please.  The purpose of this job is to run PHPUnit tests.  Hence its original name.

FAO @tebb @finnlewis